assets/src/edit-story/components/canvas/page.js

```diff
@@ -91,10 +101,11 @@ function Page() {
 			) }
 			{ displayMoveable && targetEl && (
 				<Moveable
+					ref={ moveable }
 					target={ targetEl }
-					draggable={ true }
-					resizable={ true }
-					rotatable={ true }
+					draggable={ ! selectedElements[ 0 ].isFullbleed }
```

Comment:
We can block this on the underlying data management level, e.g. as the following invariant:

* The selection object can be either a list of *only* the background element id or a list of element ids of which none are equal to the background element id.

It will be enforced as:

* If you are setting the selection list, the background element must either be the only element or will be filtered out of the list.
* If you are adding the background element to the selection list, the previous selection must be empty or nothing happens.
* If you are adding a non-background element to the selection list, and the previous selection was the background element, the new element will be the only selected element.

Would that make sense?

Reply:
SGTM.

Reply:
@barklund @pbakaus to clarify, this PR is about "fullbleed" elements, not "background" elements. Background implies fullbleed, but also has more properties, such as being the only allowed "background" element on the page, and always being the bottom layer.

Selection rules make sense to me here, but

> If you are adding the background element to the selection list, the previous selection must be empty or nothing happens.

I think the last user action should win. I.e. when background element is added to a non-empty selection, the previously selected elements should be unselected and the new selection should only contain the background element.